plugins/GSdx/GLLoader.cpp

```diff
@@ -335,12 +335,12 @@ namespace GLLoader {
 		fprintf(stdout, "OpenGL information. GPU: %s. Vendor: %s. Driver: %s\n", glGetString(GL_RENDERER), vendor, &s[v]);
 
 		// Name changed but driver is still bad!
-		if (strstr(vendor, "ATI") || strstr(vendor, "Advanced Micro Devices"))
+		if (strstr(vendor, "ATI") || strstr(vendor, "Advanced Micro Devices") || strstr(vendor, "ATI Technologies Inc."))
```

Comment:
Very minor note: It would be nicer if `"ATI"` and `"ATI Technologies Inc."` were next to each other. :P